ci-operator/step-registry/assisted/common/setup/prepare/assisted-common-setup-prepare-commands.sh

```diff
@@ -71,8 +71,8 @@ export PUBLIC_CONTAINER_REGISTRIES="{{ CI_REGISTRIES | join(',') }}"
 
 {% if ENVIRONMENT == "production" %}
 # Testing against the production AI parameters
-export PULL_SECRET='\$(cat /root/prod/pull-secret)'
-export OFFLINE_TOKEN='\$(cat /root/prod/offline-token)'
+export PULL_SECRET=\$(cat /root/prod/pull-secret)
+export OFFLINE_TOKEN=\$(cat /root/prod/offline-token)
 export REMOTE_SERVICE_URL=https://api.openshift.com
 export NO_MINIKUBE=true
 export MAKEFILE_TARGET='create_full_environment test_parallel'
```

Comment:
BTW, ``WORKER_DISK`` special manipulation for prod should be redundant now after openshift/assisted-test-infra#1518

Reply:
removed